examples/server/x11vncserver.c

```diff
@@ -0,0 +1,179 @@
+// Compile with LIBS := -lvncserver -lxcb -lxcb-xtest -lxcb-keysyms
+// Need CMake 3.24.0 to find these libraries. see https://cmake.org/cmake/help/v3.24/module/FindX11.html
```

Comment:
Ah I just saw this. The 3.24.0 FindX11 has XCB support? Please note this in CMakeLists.txt as well to inform other devs.